internal/pkg/hub/manifest.go

```diff
@@ -0,0 +1,95 @@
+package hub
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/confluentinc/cli/internal/pkg/ccstructs"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/log"
+)
+
+func (c *Client) GetRemoteManifest(owner, name, version string) (*ccstructs.Manifest, error) {
+	manifestUrl := fmt.Sprintf("%s/api/plugins/%s/%s", c.URL, owner, name)
+	if version != "latest" {
+		manifestUrl = fmt.Sprintf("%s/versions/%s", manifestUrl, version)
+	}
+
+	req, err := http.NewRequest(http.MethodGet, manifestUrl, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	if c.Debug {
+		dump, err := httputil.DumpRequestOut(req, true)
+		if err != nil {
+			return nil, err
+		}
+		log.CliLogger.Tracef("\n%s\n", string(dump))
```

Comment:
We should make an effort to standardize the logging messages- typically log files don't contain whitespace gaps.
```suggestion
		log.CliLogger.Trace(string(dump))
```

Reply:
I did this to match the style of the logging in the ccloud-sdk-go-v2 callAPI functions: `log.Printf("\n%s\n", string(dump))`.